addons/hr_evaluation/hr_evaluation.py

```diff
@@ -1,375 +1,465 @@
+
+    @api.one
+    @api.depends('state', 'interview_deadline')
+    def _get_tot_sent_appraisal(self):
+        for records in self._get_total_appraisal():
+            self.tot_sent_appraisal = len(records)
+
+    @api.one
+    def _get_tot_completed_appraisal(self):
+        for records in self._get_total_appraisal():
+            self.tot_comp_appraisal = len(records.filtered(lambda r: r.state == 'done'))
+
+    meeting_id = fields.Many2one('calendar.event', string='Meeting')
+    interview_deadline = fields.Date(string="Final Interview", select=True)
+    employee_id = fields.Many2one('hr.employee', required=True, string='Employee')
+    department_id = fields.Many2one('hr.department', string='Department')
+    evaluation = fields.Text(string='Evaluation Summary', help="If the evaluation does not meet the expectations, you can propose an action plan")
+    state = fields.Selection(EVALUATION_STATE, string='Status', track_visibility='onchange', required=True, readonly=True, copy=False, default='new', select=True)
+    date_close = fields.Datetime(string='Appraisal Deadline', select=True, required=True)
+    appraisal_manager = fields.Boolean(string='Manager')
+    appraisal_manager_ids = fields.Many2many('hr.employee', 'evaluation_appraisal_manager_rel', 'hr_evaluation_evaluation_id')
+    appraisal_manager_survey_id = fields.Many2one('survey.survey', string='Manager Appraisal', required=False)
+    appraisal_colleagues = fields.Boolean(string='Colleagues')
+    appraisal_colleagues_ids = fields.Many2many('hr.employee', 'evaluation_appraisal_colleagues_rel', 'hr_evaluation_evaluation_id')
+    appraisal_colleagues_survey_id = fields.Many2one('survey.survey', string="Employee's Appraisal")
+    appraisal_self = fields.Boolean(string='Employee')
+    appraisal_employee = fields.Char(string='Employee Name')
+    appraisal_self_survey_id = fields.Many2one('survey.survey', string='Self Appraisal')
+    appraisal_subordinates = fields.Boolean(string='Collaborator')
+    appraisal_subordinates_ids = fields.Many2many('hr.employee', 'evaluation_appraisal_subordinates_rel', 'hr_evaluation_evaluation_id')
+    appraisal_subordinates_survey_id = fields.Many2one('survey.survey', string="collaborate's Appraisal")
+    color = fields.Integer(string='Color Index')
+    display_name = fields.Char(compute='_set_display_name')
+    mail_template = fields.Many2one('mail.template', string="Email Template For Appraisal", default=_set_default_template)
+    email_to = fields.Char(string='Appraisal Receiver')
+    appraisal_url = fields.Char(string='Appraisal URL')
+    tot_sent_appraisal = fields.Integer(string='Number of sent appraisal', compute='_get_tot_sent_appraisal', default=0, store=True)
+    tot_comp_appraisal = fields.Integer(string='Number of completed appraisal', compute="_get_tot_completed_appraisal")
+    user_id = fields.Many2one('res.users', string='Related User', default=lambda self: self.env.uid)
```

Comment:
Related user ? What dose that mean ? The user that created the appraisal, the user related to the employee, anyone else ?